generator/road_access_generator.cpp

```diff
@@ -213,22 +255,49 @@ RoadAccessTagProcessor::RoadAccessTagProcessor(VehicleType vehicleType)
   }
 }
 
-void RoadAccessTagProcessor::Process(OsmElement const & elem, ofstream & oss) const
+void RoadAccessTagProcessor::Process(OsmElement const & elem, ofstream & oss)
 {
-  // todo(@m) Add support for non-way elements, such as barrier=gate.
+  // We will proccess all nodes before ways because of o5m format:
+  // all nodes are first, than all ways, than all relations.
```

Comment:
You rely on this fact. Is it a part of o5m format?

Reply:
Yes:
```
Format description
The structure of the new .o5m format is similar to the structure of conventional .osm format: Objects are stored using a strict sequence. First all nodes, then all ways, and finally all relations. Within each of these three groups, the objects are sequenced by their ids, in ascending order.
```
http://wiki.openstreetmap.org/wiki/O5m

Reply:
Ok